platform/broadcom/sai.mk

```diff
@@ -1,4 +1,4 @@
-LIBSAIBCM_VERSION = 7.1.0.0-1
+LIBSAIBCM_VERSION = 7.1.0.0-2
```

Comment:
can you only update the dnx platform?

Reply:
@lguohan have updated the saibcm only for DNX